mkdocs/docs/experiments/categories/azure/azure-disk-loss.md

```diff
@@ -0,0 +1,231 @@
+## Introduction
+
+- It causes detachment of virtual disk from an Azure instance before re-attaching it back to the instance after the specified chaos duration.
```

Comment:
Add this doc to the navigations here: https://github.com/litmuschaos/litmus/blob/master/mkdocs/mkdocs.yml#L125

Reply:
add a entry to this table also: https://github.com/litmuschaos/litmus/blob/master/mkdocs/docs/experiments/categories/contents.md#azure